packages/dcos-ui-service/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp -r "/pkg/src/dcos-ui-package" "$PKG_PATH"/usr/
```

Comment:
`[10:07:42][Building package dcos-ui-service variant <default>] cp: cannot stat '/pkg/src/dcos-ui-package': No such file or directory`

Reply:
I bet it would be 
```suggestion
cp -r "/pkg/src/dcos-ui-service" "$PKG_PATH"/usr/
```

Reply:
Thank you, change added.